src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -39,26 +41,35 @@ static constexpr std::wstring_view SCHEMES_KEY{ L"schemes" };
 std::unique_ptr<CascadiaSettings> CascadiaSettings::LoadAll(const bool saveOnLoad)
 {
     std::unique_ptr<CascadiaSettings> resultPtr;
-    std::optional<winrt::hstring> fileData = _IsPackaged() ?
-                                             _LoadAsPackagedApp() : _LoadAsUnpackagedApp();
+    std::optional<std::string> fileData = _IsPackaged() ?

```

Comment:
file data is most correctly a bucket of bytes -- perhaps we want std::vector<std::byte>? or a unique_ptr of those[]?

Reply:
Discussed online - this is probably not necessary for this PR